applications/asset_tracker_v2/src/cloud/cloud_codec/nrf_cloud/nrf_cloud_codec.c

```diff
@@ -229,11 +230,41 @@ static int add_batch_data(cJSON *array, enum batch_data_type type, void *buf, si
 int cloud_codec_encode_neighbor_cells(struct cloud_codec_data *output,
 				      struct cloud_data_neighbor_cells *neighbor_cells)
 {
+	int err;
+	char *buffer;
+	cJSON *root_obj = NULL;
+
 	__ASSERT_NO_MSG(output != NULL);
 	__ASSERT_NO_MSG(neighbor_cells != NULL);
 
-	neighbor_cells->queued = false;
-	return -ENOTSUP;
+	struct lte_lc_cells_info info = neighbor_cells->cell_data;
```

Comment:
rsrp and rsrq values are already converted here.
the nrf_cloud functions expect them to still be an index.